packages/client-node/src/connection/node_base_connection.ts

```diff
@@ -354,6 +354,40 @@ export abstract class NodeBaseConnection
     }
   }
 
+  async drainHttpResponse(stream: Stream.Readable): Promise<void> {
+    return new Promise((resolve, reject) => {
+      function dropdata() {
+        // We don't care about the data
```

Comment:
Do we need to add `resolve()` here? Also, minor nit: `dropData`

Reply:
> Do we need to add resolve() here?
No. This function will be called when data is available but we just want to throw away the data, we don't care.

Reply:
>minor nit: dropData

Fixed